Src/PerformanceCollector/Shared/Implementation/SdkVersionUtils.cs

```diff
@@ -1,16 +1,20 @@
 ﻿namespace Microsoft.ApplicationInsights.Extensibility.PerfCounterCollector.Implementation
 {
+    using System;
     using System.Linq;
     using System.Reflection;
 
     internal class SdkVersionUtils
     {
-        internal static string GetAssemblyVersion()
+        internal static string GetSdkVersion(string versionPrefix)
```

Comment:
Are there multiple copies because they are in different projects?  Is there a clean way to share one copy?